gslib/commands/config.py

```diff
@@ -990,7 +990,27 @@ def _WriteBotoConfigFile(self,
             '# To impersonate a service account for "%s://" URIs over\n'
             '# JSON API, edit and uncomment the following line:\n'
             '#%s_impersonate_service_account = <service account email>\n\n')
-      config_file.write('\n')
+
+    # Add device certificate mTLS fields.
+    config_file.write(
+        textwrap.dedent("""\
+        # This flag enables or disables mutual TLS authentication. By default,
+        # the flag is set to "false". When it is set to "true", the gsutil will
+        # use the configured client certificate as transport credential to
+        # access the APIs. The use of mTLS ensures that the access originates
```

Comment:
I don't think the user needs to worry about this, but the two APIs in play here are storage.googleapis.com and storage.mtls.googleapis.com. I don't think the first one is used if mTLS is enabled. "APIs" may also refer to the testing and staging API, but those are Google-internal.

Your new version of the paragraph seems to avoid this phrasing, so we're good.